synchronizer/views/app.py

```diff
@@ -126,6 +129,37 @@ def delete_connector(connector_id):
     return redirect(url_for("app_routes.get_connectors"))
 
 
+@app_routes.route('/connector/edit/<int:connector_id>', methods=["GET", "POST"])
+@login_required
+def edit_connector(connector_id):
+    """
+    Edits connector
+    """
+    connector = Connector.query.filter_by(id=connector_id).first()
+
+    ConnectorForm = model_form(
+        Connector,
+        base_class=FlaskForm,
+        db_session=db.session
+    )
+    form = ConnectorForm(obj=connector)
+    if form.validate_on_submit():
+        connector.name=form.name.data,

```

Comment:
`form.populate_obj(connector)` is not working? Looks like here should be a better solution.